packages/pn-commons/src/components/CustomTableRow.tsx

```diff
@@ -7,11 +7,17 @@ type Props = {
 };
 
 const CustomTableRow = ({ label, value, dataTestId }: Props) => (
-  <TableRow sx={{ '& td': { border: 'none' }, verticalAlign: 'top' }} data-testid={dataTestId}>
-    <TableCell padding="none" sx={{ py: 1 }} data-testid="label">
+  <TableRow sx={{
+    paddingBottom: { xs: '1rem' },
```

Comment:
As in the comment below, use 2 instead of 1rem

Reply:
Changed.